flathub.json

```diff
@@ -1,3 +1,3 @@
 {
-    "only-arches": ["x86_64", "i386"]
+    "only-arches": ["x86_64"]
```

Comment:
Can you split this commit out to its own pull request, so we can merge it right away?

This way we can focus this merge request on the harder things related to the update (including the PyQt bump which seems complex)